sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -1,16 +1,163 @@
 # Azure Event Hubs client library for .NET
 
-Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers. This lets you process and analyze the massive amounts of data produced by your connected devices and applications. Once Event Hubs has collected the data, you can retrieve, transform and store it by using any real-time analytics provider or with batching/storage adapters.
+Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers. This lets you process and analyze the massive amounts of data produced by your connected devices and applications. Once Event Hubs has collected the data, you can retrieve, transform and store it by using any real-time analytics provider or with batching/storage adapters.  If you would like to know more about Azure Event Hubs, you may wish to review: [What is Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about)? 
```

Comment:
One thing I noticed when opening this in VS Code was that these paragraphs are on a single line. Using tools like [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap), keeps this within the same screen for text editors and it'll be formatted the same when end users view it.

Reply:
That's intentional.  In my opinion, especially when it comes to text, it is best to allow folks to configure their editors to wrap in the way that they find it easiest to work with.   Given the range of resolutions and preferences, it's not something that I believe we can comfortably say "yes, this should only be 80 characters wide" without robbing productivity.  

For example, GitHub's native decision to limit width makes my experience undesirable since I work at a 4K resolution;  there's a ton of wasted space in my browser.

Reply:
I feel like it makes it hard during reviews, because you can't select a particular line for a paragraph, because it lives all on a single line. One link, then I'll let it go. 😆 
https://about.gitlab.com/2016/10/11/wrapping-text/

Reply:
That's a valid point, but I think that we need to start by acknowledging two things:

1) Reviews are not the majority scenario, they happen much less frequently than local edits, iterations, and reading of the content.
2) GitHub's review experience is... very not good... in my opinion.  There is a lot to be desired.

I really don't want to optimize for reviews over the experience of working with the contents in a normal development flow.

Reply:
What I usually do when reviewing markdown files is one of the following
- If in Github, open that particular file separately (there is a view file button), use that to read, have the original view open as well for comments
- If using VS Code and I have checked out the branch, then use the Markdown Preview to see the rendered text
- If using VS Code and Github Pull Request Extension, then add comments in VS Code itself